### PR TITLE
chore: updating go version from 1.19 to 1.20

### PR DESCRIPTION
**Pull request by gsk967**
## Description

closes: #XXXX

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by robert-zaremba** [comment]
Note: this should be merged only after v4.3 is released.

File: tests/simulation/state.go [hunk 1]
```diff
@@ -185,12 +185,8 @@ func appStateFn(cdc codec.JSONCodec, simManager *module.SimulationManager) simty
...
 		for _, val := range stakingState.Validators {
 			if val.Status != stakingtypes.Bonded {
-				_, _, ethAddr, err := util.GenerateRandomEthKeyFromRand(r)
-				if err != nil {
-					panic(err)
-				}
-
-				gravityEthAddr, err := gravitytypes.NewEthAddress(ethAddr.Hex())
+				dummyethaddr := hex.EncodeToString([]byte(val.OperatorAddress))
+				gravityEthAddr, err := gravitytypes.NewEthAddress("0x" + dummyethaddr[:40])
```
[the remark this answers is not in the document]
Comment (gsk967) [reply tests/simulation/state.go:189]: Yes , 
https://github.com/umee-network/umee/blob/main/tests/util/ethereum.go#L21
```
func GenerateKey(c elliptic.Curve, rand io.Reader) (*PrivateKey, error) {
	randutil.MaybeReadByte(rand)
	...
}

```
https://cs.opensource.google/go/go/+/refs/tags/go1.20.3:src/crypto/ecdsa/ecdsa.go;l=154

[the remark this answers is not in the document]
Comment (gsk967) [reply tests/simulation/state.go:189]: https://cs.opensource.google/go/go/+/refs/tags/go1.20.3:src/crypto/ecdsa/ecdsa.go;l=155

**Review by robert-zaremba — APPROVED** [review]

**Comment by robert-zaremba** [comment]
Please update RELEASE NOTES and CHANGELOG

**Pull request merged**